### PR TITLE
perf(aad-manifest): show learn more button when deploy aad success

### PR DESCRIPTION
**Pull request by SLdragon**
Address this bug: [Bug 14177210](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14177210): Update wording/notification for deploy AAD manifest

image
image
image

### CODE REVIEW

File: packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts [hunk 1]
```diff
@@ -223,7 +231,18 @@ export async function deploy(
             }
           });
       } else {
-        ctx.userInteraction.showMessage("info", msg, false);
+        if (isDeployAADManifestFromVSCode) {
+          ctx.userInteraction
+            .showMessage("info", msg, false, getLocalizedString("core.deploy.aadManifestLearnMore"))
```
Comment (KennethBWSong) [inline packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts:236]: What will happen in cli? Will there be a button to click in cli?

Comment (KennethBWSong) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts:236]: Will have follow up after offline discussion.

**Review by KennethBWSong — APPROVED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts:236]: This logic is only for VSCode, for CLI part, I will create a task and discuss with Junjie how to update it in the future PR.

**Review by nliu-ms — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Pull request merged**